radicle-node/src/runtime/handle.rs

```diff
@@ -136,10 +136,10 @@ impl<G: Signer + Ecdh + 'static> radicle::node::Handle for Handle<G> {
         receiver.recv().map_err(Error::from)
     }
 
-    fn tracked_repos(&self) -> Result<Self::TrackedRepos, Self::Error> {
+    fn repo_policies(&self) -> Result<Self::TrackedRepos, Self::Error> {
```

Comment:
And the associated types too
```suggestion
    fn repo_policies(&self) -> Result<Self::RepoPolicies, Self::Error> {
```